scripts/Player.gd

```diff
@@ -5,6 +5,7 @@ class_name Player
 @export var weapon_list : Dictionary
 
 var move_direction : Vector2 = Vector2.ZERO
+@onready var shoot_vfx: CPUParticles2D = $ShootVFX
```

Comment:
Is it shooting particle or sound particle ?

Reply:
is "emitting sound" particle.
maybe something like `sound_shoot_vfx` name is better ?

Reply:
Yeah I think so